src/connectivity/mod.rs

```diff
@@ -971,7 +971,7 @@ pub fn local_complement(
 pub fn graph_all_simple_paths(
     graph: &graph::PyGraph,
     origin: usize,
-    to: usize,
+    to: &Bound<PyAny>,
```

Comment:
Use the enum I described #1487

This way, handling the cases is a simple `match` statement shared by both methods

Reply:
Fixed in the new commit